Misc/NEWS.d/next/Library/2024-10-30-23-42-44.gh-issue-126223.k2qooc.rst

```diff
@@ -0,0 +1,2 @@
+Fixed :exc:`SystemError` upon calling :func:`!_interpreters.create` with
+an invalid unicode character.
```

Comment:
```suggestion
Raise a :exc:`UnicodeEncodeError` instead of a :exc:`SystemError` upon
calling :func:`!_interpreters.create` with an invalid Unicode character.
```

This *might* be a user-facing change, so maybe a What's New entry is needed?

Reply:
`_interpreters` is technically private, but I've been considering it public in my PRs because it affects Eric's PEP 734 package.